src/transform.ts

```diff
@@ -82,6 +81,9 @@ export function typeToString(def: TypeDef): string {
     }
     return type;
   }
+  if (Array.isArray(type)) {
+    return type.map((elem) => typeToString({ type: elem })).join(' | ');
```

Comment:
This only works because you don't have function or constructor types.  Otherwise e.g. transforming from an array of `(a: number) => string` and `number` gives the type `(a: number) => string | number`, aka `(a: number) => (string | number)`, instead of the intended `((a: number) => string) | (number)`.
Please add the brackets -- even though it cannot happen right now.

Reply:
Of course, constructors and functions are not a part of json schema.
I can add if you think this is important